app/css/openmrs-owa-platformui.css

```diff
@@ -7,236 +7,273 @@
  * graphic logo is a trademark of OpenMRS Inc.
  */
 body {
-    font-family: "OpenSans", Arial, sans-serif;
-    -webkit-font-smoothing: subpixel-antialiased;
-    margin: 10px auto;
-    background: #eeeeee;
-    color: #363463;
-    font-size: 16px;
+  font-family: "OpenSans", Arial, sans-serif;
+  -webkit-font-smoothing: subpixel-antialiased;
+  margin: 10px auto;
+  background: #eeeeee;
+  color: #363463;
+  font-size: 16px;
+  max-width: 1000px;
 }
 
 header {
-    position: relative;
-    background-color: #00463f;
-    color: #CCC;
-    line-height: 1em;
-    -moz-border-radius: 5px;
-    -webkit-border-radius: 5px;
-    -o-border-radius: 5px;
-    -ms-border-radius: 5px;
-    -khtml-border-radius: 5px;
-    border-radius: 5px;
-    display: block;
-    min-height: 48px;
+  position: relative;
+  background-color: #00463f;
+  color: #ccc;
+  line-height: 1em;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -o-border-radius: 5px;
+  -ms-border-radius: 5px;
+  -khtml-border-radius: 5px;
+  border-radius: 5px;
+  display: block;
+  min-height: 48px;
+  margin-bottom: 40px;
 }
 
 header .logo {
-    float: left;
-    margin: 4px;
-    display: block;
+  float: left;
+  margin: 4px;
+  display: block;
 }
 
 header .logo img {
-    width: 120px;
+  width: 120px;
 }
 
 #body-wrapper {
-    margin-top: 10px;
-    padding: 10px;
-    background-color: white;
-    -moz-border-radius: 5px;
-    -webkit-border-radius: 5px;
-    -o-border-radius: 5px;
-    -ms-border-radius: 5px;
-    -khtml-border-radius: 5px;
-    border-radius: 5px;
-    min-height: 500px;
-    justify-content: center;
+  margin-top: 10px;
+  padding: 10px;
+  background-color: white;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -o-border-radius: 5px;
+  -ms-border-radius: 5px;
+  -khtml-border-radius: 5px;
+  border-radius: 5px;
+  min-height: 500px;
+  justify-content: center;
 }
 
 .app-title {
-    display: block;
-    float: right;
-    color: #eeeeee;
-    font-size: 32px;
-    margin: 4px;
-    height: 100%;
-    padding-top: 12px;
-    padding-right: 5px;
+  display: block;
+  float: right;
+  color: #eeeeee;
+  font-size: 32px;
+  margin: 4px;
+  height: 100%;
+  padding-top: 12px;
+  padding-right: 5px;
 }
 
 .instructions {
-    align-self: center;
+  align-self: center;
 }
 
-
 /* Custom CSS Starts here */
 
-html, body {
-    height: 100%;
-    margin: 0;
-    padding: 0;
-    background-color: #ffffff;
-  }
-  .main-container {
-    top: 60px;
-  }
-  .login-header {
```

Comment:
Is this a change in formatting?

Reply:
Yes, it is a formatting change. The spaces were unnecessary.

Reply:
Can we do the formatting using a separate pull request? That way it will be easier to tell what exactly changed other than formatting.

Reply:
@dkayiwa, thank you for that recommendation. I have updated the CSS file to only show the file changes. I will format the file in another PR and ticket.